hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AAnalyticsAcceleratorStreamReading.java

```diff
@@ -64,8 +64,6 @@
 public class ITestS3AAnalyticsAcceleratorStreamReading extends AbstractS3ATestBase {
 
   private static final String PHYSICAL_IO_PREFIX = "physicalio";
```

Comment:
we can get rid of this   private static final String PHYSICAL_IO_PREFIX = "physicalio"; too right?

Reply:
no it's used in the last test